pkgs/desktops/gnome/apps/fragments/dependency-resolution.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/meson.build b/meson.build
+index 5030d0c..27d41e5 100644
+--- a/meson.build
++++ b/meson.build
+@@ -32,10 +32,11 @@ transmission_dep = declare_dependency(
+ 				meson.get_compiler('c').find_library('dht'),
+ 				meson.get_compiler('c').find_library('natpmp'),
+ 				meson.get_compiler('c').find_library('event'),
+-				meson.get_compiler('c').find_library('libcurl'),
+-				meson.get_compiler('c').find_library('libcrypto'),
++				meson.get_compiler('c').find_library('libcurl', dirs: '@curl@/lib'),
```

Comment:
`find_library` should be passed library name without `lib` prefix.

Reply:
This worked for curl, but it still doesn't detect openssl and zlib. Would you have any advice?

Reply:
What did you try? The patch now says `find_library('libcurl')` instead of `find_library('curl')`. Similarly, it should be `find_library('z')`.

Reply:
Ah that cleared it up, thanks!